js/advice-feedback.js

```diff
@@ -0,0 +1,49 @@
+const initAdviceFeedback = () => {
+    try {
+        const d = document;
+        const form = d.getElementById('cads-advice-feedback');
+        form.classList.add('step1');
+        const rg = d.getElementsByName('radiogroup-advice-feedback');
+
+        rg[0].addEventListener('change', () => {
+            // Yes
+            // form.classList.remove('step1');
+            form.classList.add('step3');
+
+            // TODO Send respong to API
+            console.log('Send "page was useful" reply');
+        });
+
+        rg[1].addEventListener('change', () => {
+            // No
+            // form.classList.remove('step1');
```

Comment:
Presumably this is commented out for a reason?

Reply:
To be decided if we show/hide the other steps.